configure.ac

```diff
@@ -818,13 +819,25 @@ AC_ARG_ENABLE([openvpn],
     [ENABLED_OPENVPN=$enableval],
     [ENABLED_OPENVPN=no])
 
+# openresty compatibility build
+AC_ARG_ENABLE([openresty],
+    [AS_HELP_STRING([--enable-openresty],[Enable openresty (default: disabled)])],
+    [ ENABLED_OPENRESTY=$enableval ],
+    [ ENABLED_OPENRESTY=no ]
+    )
+
 # nginx compatibility build
 AC_ARG_ENABLE([nginx],
     [AS_HELP_STRING([--enable-nginx],[Enable nginx (default: disabled)])],
     [ ENABLED_NGINX=$enableval ],
     [ ENABLED_NGINX=no ]
     )
 
+if test "$ENABLED_OPENRESTY" = "yes"
+then
+    ENABLED_NGINX="yes"
```

Comment:
So the openresty also enables NGINX because of the compatibility API's it enables? Is there another way we can group these by openssl feature block or version?